### PR TITLE
[TR] PIM-3997: fix ci error on EE

### PR DESCRIPTION
**Pull request by sumbobyboys**
| Q                    | A
| -------------------- | ---
| Bug fix?             | Y
| New feature?         | N
| BC breaks?           | N
| CI currently passes? | Y
| Tests pass?          | Y
| Scenarios pass?      | Y
| Checkstyle issues?*  | N
| PMD issues?**        | N
| Changelog updated?   | N
| Fixed tickets        | PIM-3997
| DB schema updated?   |
| Migration script?    |
| Doc PR               |

### CODE REVIEW

File: src/Pim/Bundle/CatalogBundle/Command/GetProductCommand.php [hunk 1]
```diff
@@ -61,4 +73,45 @@ protected function getProduct($identifier)
...
+     */
+    public function getSecurityFacade()
+    {
+        return $this->getContainer()->get('oro_security.security_facade');
+    }
+
+    /**
+     * Create a security token from the given username
+     *
+     * @param OutputInterface  $output
```
Comment (grena) [inline src/Pim/Bundle/CatalogBundle/Command/GetProductCommand.php]: Extra space `OutputInterface  $output`

**Comment by grena** [comment]
GTM once little comment fixed!

**Comment by jmleroux** [comment]
GTM

File: src/Pim/Bundle/CatalogBundle/Command/GetProductCommand.php [hunk 2]
```diff
@@ -44,6 +51,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return;
         }
 
+        $username = $input->getArgument('username');
+        if (!$this->createToken($output, $username)) {
+            return;
```
Comment (solivier) [inline src/Pim/Bundle/CatalogBundle/Command/GetProductCommand.php]: return null;

Comment (sumbobyboys) [inline src/Pim/Bundle/CatalogBundle/Command/GetProductCommand.php]: No need to add null here, it doesn't make sens

**Pull request merged**